include/mbedtls/config.h

```diff
@@ -275,6 +275,8 @@
 //#define MBEDTLS_SHA1_ALT
 //#define MBEDTLS_SHA256_ALT
 //#define MBEDTLS_SHA512_ALT
+//#define MBEDTLS_DHM_ALT
```

Comment:
This should be placed so that the list of alternative options is in alphabetical order.

Reply:
OK